ai-platform-unified/notebooks/official/gapic/vertex-matching-engine-for-indexing.ipynb

```diff
@@ -0,0 +1,1368 @@
+{
```

Comment:
<p>Please incorporate this link: https://cloud.google.com/ai-platform-unified/docs/general/vpc-peering</p><p> </p><p>(I will update the link later on, but for now I'd like it to be included here)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>